pkgs/applications/audio/pulseaudio-dlna/default.nix

```diff
@@ -14,24 +31,34 @@ assert flacSupport -> flac != null;
 assert soxSupport -> sox != null;
 assert vorbisSupport -> vorbis-tools != null;
 
```

Comment:
```suggestion
```

Reply:
Why removing this two and not the rest?

Reply:
Because github did not allow me to do that :P.

Reply:
Addressed in 03edc17